pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -534,6 +534,97 @@ var _ = Describe("Converter", func() {
 			vmi.Spec.Networks = append(vmi.Spec.Networks, *net)
 			Expect(Convert_v1_VirtualMachine_To_api_Domain(vmi, &Domain{}, c)).ToNot(Succeed())
 		})
+
+		It("should add tcp if protocol not exist", func() {
```

Comment:
nit: my understanding is that we should define separate Contexts if we target different code. Test cases above are for convertion functions; you added several cases specifically for configPortForward; I think it would make sense to have them separate in a new Context.

Reply:
No problem I will move it to a new context